workspaces/frontend/package.json

```
@@ -94,10 +94,13 @@
     "webpack-merge": "^5.10.0"
   },
   "dependencies": {
+    "@patternfly/react-code-editor": "^6.1.0",
```

Probably best to keep it at `6.0.0` atm to align with the other PatternFly dependencies.
```suggestion
    "@patternfly/react-code-editor": "^6.0.0",
```